packages/patternfly-4/react-charts/src/components/BarChart/examples/common/getContainerProps.js

```diff
@@ -14,10 +14,6 @@ const styles = StyleSheet.create({
       '.chart-overflow': {
         ':not(foo) svg': {
```

Comment:
Not needed now, but we can actually replace these with '& svg' the not food was a work around for adding child selectors with Aphrodite. React-styles is now using emotion which supports it out of the box.

Reply:
Thanks, I've made the change.